### ISSUE TITLE
PipeChainStart with custom pipe operators

### ISSUE THREAD

**Issue by fuelen** [rrrene/credo#828]
### Environment

* Credo version (`mix credo -v`): 1.5.1
* Erlang/Elixir version (`elixir -v`): 1.11.2

### What were you trying to do?
I have custom implementation for operator `~>`. It is used with regular `|>` this way
```elixir
defmodule CredoTest do
  defmacro a ~> b do
    quote do
      unquote(a) |> unquote(b)
    end
  end

  def test do
    1
    |> Kernel.*(2)
    ~> Kernel.*(2)
    |> Kernel.*(2)
  end
end
```

### Expected outcome
I find my piece of code readable, it would be great to add a list of allowed operators to config too.
I tried `excluded_functions: [:~>]` but this didn't work.

### Actual outcome
Credo.Check.Refactor.PipeChainStart is triggered

**Comment by rrrene** [comment]
Thx for reporting. This is a duplicate of #740 :+1:
